index.html

```diff
@@ -1,48 +1,56 @@
+              </path>
             </svg>
           </a>
         </div>
         <nav class="navbar-collapse collapse keen-navbar-collapse" role="navigation">
-          <ul class="nav navbar-nav main-nav">
-            <li><a href="https://keen.io/blog/101269629091/charts-on-grids-responsive-dashboard-templates-with?s=db_land" target="_blank">About</a></li>
+          <ul class="navbar-nav nav main-nav">
+            <li><a
+                href="https://keen.io/blog/101269629091/charts-on-grids-responsive-dashboard-templates-with?s=db_land"
+                target="_blank">About</a></li>
             <li><a href="https://github.com/keen/dashboards" target="_blank">Docs</a></li>
             <li><a href="https://keen.io/team?s=db_land" target="_blank">Team</a></li>
             <li><a href="https://blog.keen.io/?s=db_land" target="_blank">Blog</a></li>
```

Comment:
let's remove get param `?s=db_land` - to make this link work

Reply:
let's add link:
`Charts` to https://keen.io/docs/visualize/common-chart-examples/